bindings/python/iree/compiler/core.py

```diff
@@ -203,8 +203,6 @@ def build_compile_command_line(input_file: str, tfs: TempFileSaver,
     cl.append("--iree-vm-bytecode-module-strip-debug-ops")
   if options.strip_source_map:
     cl.append("--iree-vm-bytecode-module-strip-source-map")
-  if options.strip_symbols:
-    cl.append("--iree-vm-bytecode-module-strip-symbols")
```

Comment:
Can you also remove the other references to this `strip_symbols` option in this file?

here's one:
https://github.com/google/iree/blob/10f8b24b5fc21cc93641d2cd529bd3c587d66815/bindings/python/iree/compiler/core.py#L113-L114
also used here:
https://github.com/google/iree/blob/10f8b24b5fc21cc93641d2cd529bd3c587d66815/bindings/python/tests/compiler_core_test.py#L162